### DISCUSSION TITLE
Cannot import httpexpect

### DISCUSSION THREAD

**Discussion by lstep** [gavv/httpexpect#134] [Q&A] [answered]
When getting the package, I get the following error : 
```
$ go get gopkg.in/gavv/httpexpect.v1
# gopkg.in/gavv/httpexpect.v1
../../../gopkg.in/gavv/httpexpect.v1/reporter.go:39: r.backend.FailNow undefined (type *require.Assertions has no field or method FailNow)
```
I'm using Go `v1.7.3` on `linux/amd64`.

**Comment by lstep** [comment]
I erased all traces of it in my local packages and then reinstalled, it worked. Strange...

**Comment by gavv** [comment, marked as answer]
I guess you had an outdated version of [testify](https://github.com/stretchr/testify).
